### ISSUE TITLE
About Opencvsharp, a thought from blaisexen

### ISSUE THREAD

**Issue by blaisexen** [shimat/opencvsharp#627]
@shimat 

the last version that support external files(opencv) was version 3.1 (05 Nov., 2016),
if opencv_World310 was build with Cuda then OpencvSharp.Gpu.GpuMat would be used.
this release:
https://github.com/shimat/opencvsharp/releases/tag/3.1.0.20161105

OpencvSharpExtern.DLL holds(embedded opencv libs) so we don't need opencv_worldXXX file externally anymore.
started from 3.2 (07 Jan., 2017) Support OpenCV 3.2 with contrib modules
https://github.com/shimat/opencvsharp/releases/tag/3.2.0.20170107

the latest released binaries was Opencvsharp4.0.0, I used the binaries and there was still "OpencvSharp.Cuda" namespace.

so that means cuda is still supported from the binaries released.
https://github.com/shimat/opencvsharp/releases/tag/4.0.0.20181130

I understand that you stop supporting cuda features,

BUT from the source, if we clone or download the files, then we compile, it doesn't support now, 
because when I compiled all the binaries and used it, "OpencvSharp.Cuda" context is not available.

so it means what you said that you don't support issues regarding cuda because 
the source code files released are not supporting cuda anymore.

I hope I'm right, I'm closing it now,
thank you

**Comment by blaisexen** [comment]
the file structure I meant "was asking" if OpencvSharp only uses root directory, because emgucv enters x64 folder to call the Extern file for cuda features.

**Comment by blaisexen** [comment]
@shimat

hi,
Sorry for bothering you,

But I have something to tell, surprisingly I was able to run OpenCvSharp ( with openCv3.4.5 from OpenCvSharp _3.1.0 ) with Cuda_10, I replaced/used OpenCv_World310 to OpenCv_World345 and "openCv2" include files. Now runs

But, I'm not sure if I really get it right running through....because when I try to use GPU software I see the sensor high uses the GPU but only for 5 seconds then back to normal Ghz.

Last question, 
What OpenCvSharp last/latest version source code will have namespace contain .Cuda if we compile it?
because 4.0.0 does not contain it anymore.

thanks for this wonderful software.
